CHANGES.txt

```diff
@@ -62,8 +62,10 @@ Changes
     SELECT id FROM t2
 
 - Added new "password" authentication method which is available for connections
-  via the Postgres Protocol. This method allowes clients to authenticate using
-  a valid database user and its password.
+  via the Postgres Protocol and HTTP. This method allows clients to authenticate
+  using a valid database user and its password. For HTTP the ``X-User`` header,
+  used to provide a username, has been deprecated in favor of the the standard
+  HTTP Basic Auth header ``Authorization``.
```

Comment:
... HTTP authorization header `Authorize` with the [Basic Authentication Scheme](https://tools.ietf.org/html/rfc1945#section-11.1).